octobot_commons/async_job.py

```diff
@@ -0,0 +1,127 @@
+# pylint: disable=W0703,R0902,R0913
+#  Drakkar-Software OctoBot-Commons
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import asyncio
+
+import time
+
+from octobot_commons.asyncio_tools import wait_asyncio_next_cycle
+from octobot_commons.logging.logging_util import get_logger
+
+
+class AsyncJob:
+    """
+    Async job management
+    """
+
+    NO_DELAY = -1
+
+    def __init__(
+        self,
+        callback,
+        execution_interval_delay=NO_DELAY,
```

Comment:
Means by default the scheduler is waken up at each loop of the async event loop ?

Reply:
Yes, I didn't know what to use as default value

Reply:
I think something like 0.1 would be better to avoid spam calls

Reply:
:+1: